pkgs/development/python-modules/ploomber-extension/default.nix

```diff
@@ -40,6 +40,15 @@ buildPythonPackage rec {
     pytest-jupyter
   ];
 
+  disabledTests = lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
Shouldn't this technically be `buildPlatform` given the circumstances under which these tests fail?

Reply:
Done

Reply:
I will note that even my own patches for sandbox issues just do `stdenv.isDarwin`… probably incorrectly, but it’s not like you’d ever be in a situation where buildPlatform == Darwin, hostPlatform != Darwin, *and*, doCheck = true.

So really it doesn’t matter tbh, but given you were explicitly specifying stdenv.hostPlatform, I figured I’d mention it. :)